include/kitepp/userconstants.hpp

```diff
@@ -55,6 +55,7 @@ const string VARIETY_REGULAR = "regular";
 const string VARIETY_BO = "bo";
 const string VARIETY_CO = "co";
 const string VARIETY_AMO = "amo";
+const string VERIETY_ICEBERG = "iceberg";
```

Comment:
This might be a typo

Reply:
Also, I think the commit ([dd7e06b](https://github.com/zerodha/cppkiteconnect/pull/60/commits/dd7e06bdd88929adfa1acdafaf6a319af17976b7)) is showing up so the branch might need to be rebased.

Reply:
Fixed the typo.

That commit is showing up because I had created this branch from previous PR branch (because I needed those changes here). It doesn't matter because I intend to squash the commits.

Reply:
Fixed the typo.

That commit is showing up because I had created this branch from previous PR branch (because I needed those changes here). It doesn't matter because I intend to squash the commits.